packages/shared/lib/time.ts

```diff
@@ -0,0 +1,45 @@
+import { get, writable } from 'svelte/store'
+
+export const MILLISECONDS_PER_SECOND = 1000
+export const SECONDS_PER_MINUTE = 60
+export const MINUTES_PER_HOUR = 60
+export const HOURS_PER_DAY = 24
+export const DAYS_PER_WEEK = 7
+export const MONTHS_PER_YEAR = 12
+
+/**
+ * The last timestamp that the app user was active
+ */
+export const lastActiveAt = writable<Date>(new Date())
```

Comment:
If we are refactoring the idle util functions into a separate file than the svelte component then it should be separate to the time constants.

Reply:
I agree, but until the rest of the code is like this then we should keep it consistent I think.